docker/tests/check-code.sh

```diff
@@ -1,8 +1,13 @@
-#!/bin/sh
+#!/bin/bash
 
 set -o errexit
-set -o nounset
```

Comment:
don't think this is needed & it causes issues if $1 is unset (flag isn't passed)